src/main/java/ymfc/parser/Parser.java

```diff
@@ -151,7 +151,7 @@ public static AddCommand parseRecipeSaveLine(String saveRecipeString) throws Inv
         String args = m.group("args") == null ? "" : m.group("args").trim();
 
         if (!command.equals("add")) {
-            throw new InvalidSaveLineException("Invalid command syntax found in Recipe save file: "
+            throw new InvalidSaveLineException("Invalid line found in Recipe save file: "
                     + saveRecipeString + System.lineSeparator());
```

Comment:
I'm assuming you want to indent the exception error messages to be inline with the rest of the application, but do you think other exceptions like these should also have the identation?

Reply:
Wait this will run at startup, I don't think we ever made an indent for that, since it will just blend in before everything else like it used to.

Reply:
It's fine